webapi/src/main/scala/org/knora/webapi/exceptions/Exceptions.scala

```diff
@@ -354,6 +354,9 @@ object InconsistentRepositoryDataException {
     InconsistentRepositoryDataException(message, Some(ExceptionUtil.logAndWrapIfNotSerializable(e, log)))
 }
 
+case class MissingLastModificationDateOntologyException(message: String, cause: Option[Throwable] = None)
```

Comment:
I don't know if this makes sense, but could this be renamed to something like `MissingRequiredDataInOntologyException` so we can reuse it for other cases like that (namely for missing label and missing attachedToProject attributes)?

Reply:
I depends. The error type in this case is used as part of a workflow, i.e., shutting down dsp-api in the application actor. If we start reusing types, then you will need to peak inside to know what it actuallly is. I think that the best practice is to have a type for every single error.